route/account/_account-id/token/index.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+module.exports = (server, path, options) => {
+    return options.container.call((readBodyBufferMiddleware, tokenManager, validateRequestQueryMiddleware, validateSignatureMiddleware) => {
+        return {
+            name: "account-token-create",
+            post: [
+                readBodyBufferMiddleware(),
+                validateSignatureMiddleware(),
+                validateRequestQueryMiddleware("/account/token-create-request.json"),
+                (req, res, next) => {
+                    var publicFlag;
+
+                    publicFlag = false;
+
+                    // Query string values are always strings - compare as such
+                    if (req.query && req.query.public === "true") {
```

Comment:
All incoming query values are strings.  I can't compare against a boolean or anything else.

The values of the string are laid out in the schema.  See schema/account/token-create-request.json for details, but the string is limited to "true" and "false".  It is optional and acts like "false" when not specified.